README.md

```diff
@@ -149,11 +149,21 @@ git machete advance
 Useful for merging the child branch to the current branch in a linear fashion (without creating a merge commit).
 
 ### GitHub integration
+
+Annotate the branches with GitHub PR numbers: <br/>
+```shell script
+git machete hub anno-prs
+```
+
+Check out PR and reflect its parent (grandparent, etc.) in `.git/machete`: <br/>
```

Comment:
Whoops as discussed on video, that's not gonna be implemented on the this PR... actually if you open a separate issue for implementing `checkout-pr` (however it's gonna be called)